src/Symfony/Component/Lock/Store/PostgreSqlStore.php

```diff
@@ -91,9 +91,7 @@ public function save(Key $key): void
                 return;
             }
         } finally {
-            if (!$lockAcquired) {
-                $this->getInternalStore()->delete($key);
-            }
+            $this->getInternalStore()->delete($key);
```

Comment:
This is not a useless check. `lockAcquired` might by `true`.

Reply:
I indeed used some automated tooling, but checked every single change of course. I think it's me that missed the behavior of `finally` with `return`. Anyway, you're right, I reverted this bit